app/build.gradle

```diff
@@ -31,10 +32,22 @@ android {
 }
 
 dependencies {
+    /*
+    changed the configuration of dependencies to use annotationProcessor.
+    If a dependency includes components that also need to be on the compile classpath,
+    declare that dependency a second time and use the compile dependency configuration.
+    android-developers   link
+    https://developer.android.com/studio/build/gradle-plugin-3-0-0-migration.html#annotationProcessor_config
+    */
+    annotationProcessor "com.android.support:appcompat-v7:$support_version"
     compile "com.android.support:appcompat-v7:$support_version"
+    annotationProcessor "com.android.support:design:$support_version"
     compile "com.android.support:design:$support_version"
+    annotationProcessor "com.jakewharton:butterknife:$butterknife_version"
     compile "com.jakewharton:butterknife:$butterknife_version"
-    apt "com.jakewharton:butterknife-compiler:$butterknife_version"
-
+    annotationProcessor "com.jakewharton:butterknife-compiler:$butterknife_version"
+    compile "com.jakewharton:butterknife-compiler:$butterknife_version"
+    compile "com.google.dagger:dagger-compiler:2.0"
```

Comment:
Dagger is not used in the app. Can you remove?

Reply:
I have made the changes and commited
Pls see